full-moon/src/ast/parsers.rs

```diff
@@ -8,6 +8,10 @@ use super::{
     span::ContainedSpan,
     Expression, FunctionBody, Parameter,
 };
+
+#[cfg(any(feature = "cfxlua", feature = "luau"))]
+use super::Var;
```

Comment:
Looks like this was introduced by your IDE.

You should reference it through `ast::Var` instead, which is already exported